conf/retention.go

```diff
@@ -145,11 +148,20 @@ func ParseRetentions(defs string) (Retentions, error) {
 			}
 			retention.NumChunks = uint32(i)
 		}
-		retention.Ready = true
 		if len(parts) == 5 {
-			retention.Ready, err = strconv.ParseBool(parts[4])
-			if err != nil {
-				return nil, err
+			// 0 (default) is effectively the same as 'true'
```

Comment:
Need to clarify that the ready field used to be true|false.  Otherwise, this comment won't make any sense when read in 6months time.